content/news/2024-05-14-everse-workshop/index.md

```diff
@@ -0,0 +1,39 @@
+end_date: 2024-05-14T00:00:00Z
+summary: >
+    The main goal of the workshop is to serve as a platform for EVERSE to showcase the work of the project, and receive input from all relevant stakeholders, while facilitating networking and collaboration opportunities between all participants. Stakeholders of particular relevance in the workshop are Funders and Policy Makers, in order to increase awareness on the challenges and opportunities around research, as well as relevant community projects. The expectation is for participants to bring with them their communities’ best practices and services on research software (including but limited tos quality, management, sustainability, excellence, etc) in order to engage with EVERSE, while also establishing a bidirectional channel of communication moving forward.
+---
+
+EVERSE workshop
+Satellite event to the
+**International Research Software Funders Workshop**
+
+{{< figure src="/feature.png" >}}
+
+We are thrilled to announce our upcoming **EVERSE** workshop, to be held as a satellite event to the **International Research Software Funders Workshop** , and hosted by **SciLifeLab**, in **Uppsala, Sweden.**
+Save the date: **Tuesday, September 10th.** Workshop is planned as a **full day (9 am – 5 pm CEST**, provisional agenda available [here](EVERSE_Workshop_IRSFW_2024_provisional_Agenda.pdf))
+
+The main goal of the workshop is to serve as a platform for EVERSE to showcase the work of the project, and receive input from all relevant stakeholders, while facilitating networking and collaboration opportunities between all participants. Stakeholders of particular relevance in the workshop are Funders and Policy Makers, in order to increase awareness on the challenges and opportunities around research, as well as relevant community projects. 
+The expectation is for participants to bring with them their communities’ best practices and services on research software (including but limited tos quality, management, sustainability, excellence, etc) in order to engage with EVERSE, while also establishing a bidirectional channel of communication moving forward.
+- Participation to the event is free, but registration is required. Deadline for registration is **Friday, May 31st 2024.** There is also a **limited number of places for in person participation** (max 25), so registration will be on a first-come first-serve basis.
+- Register here: EVERSE - funders forum workshop Tickets, Tue, Sep 10, 2024 at 9:30 AM | Eventbrite
+
+You can find the draft agenda of the main event of the International Research Software Funders Workshop **here**, also linked in the **event page**. Please note this workshop is open to funders of research software, open science, and open-source research software, as well as a few guests. If you are interested in attending but have not received an invitation, please contact us at res-soft-forum-uppsala@googlegroups.com or info@researchsoft.org (if you encounter issues with the first email).
```

Comment:
Shouldn't "here" and "event page" be links?

Reply:
Also, the last sentence should be rephrased to "please contact us at res-soft-forum-uppsala@googlegroups.com (or at info@researchsoft.org, if you encounter issues with the first email)."  - slightly moved the parenthesis and changed "us" to "them".